conda/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "zppy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  #sha256: ab6dee537f8252a9f244f01b725046dd64748d68
+  path: ..
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+  entry_points:
+    - zppy = zppy.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - configobj=5.0.6
+    - jinja2=2.11.3
```

@forsyth2, while it is okay (but not my preferred practice) to pin these in the development environment, we will have major issues if we pin them this precisely in the conda recipe.  There should only be constraints on these packages if there are known incompatibilities with past versions or expected incompatibilities with future versions.  This depends on knowing the packages themselves and their histories.

I happen to know that `jinja2` 3.0.0 (why not `jinja3`, I can't say) was released a few days ago.  I would therefore expect that it might be important to test that `jinja2` 3.0.0 works well or do `jinja2 <3.0.0`.

conda-forge is picky about linting recipes.  If you want to pin a specific version, you either need no `=` or a ` ==` (note the space).

```suggestion
    - configobj 
    - jinja2
```
or
```suggestion
    - configobj >=5.0.0,<6.0.0
    - jinja2 >=2.0.0,<3.0.0
```

---

I looked into `configobj` and it hasn't been updated since 2014, which is alarming.  But that does suggest that pinning to the latest version will probably do no harm.